README.md

```diff
@@ -9,3 +9,7 @@ The launcher contains a search bar but very few app
 the intent this search bar sends.
 So this app allows the search bar to launch a search in your browser
 instead of doing nothing.
+
+[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
+     alt="Get it on F-Droid"
+     height="80">](https://f-droid.org/packages/pro.rudloff.search_to_browser/)
```

Comment:
Using HTML in Markown is not a very good practice. Can you please use pure Markdown?